utils/sn_flux_est.py

```diff
@@ -54,17 +90,24 @@ def plot_flux_estimation(nu_atnf, S_atnf, S_atnf_e, my_nu, my_S, my_S_e, a, K, c
     for element in S_cont:
         S_range.append(element)
 
+    fig, ax = plt.subplots(figsize=(12, 8))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused variable 'fig'](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4635257)